test/Microsoft.Azure.Devices.Edge.Test.Common/Windows/EdgeDaemon.cs

```diff
@@ -23,12 +17,17 @@ public EdgeDaemon(string scriptDir)
             this.scriptDir = scriptDir;
         }
 
-        public Task InstallAsync(
-            string deviceConnectionString,
-            Option<string> packagesPath,
-            Option<Uri> proxy,
-            CancellationToken token)
+        public Task InstallAsync(string deviceConnectionString, Option<string> packagesPath, Option<Uri> proxy, CancellationToken token)
         {
+            var properties = new object[] { };
+            string message = "Installed edge daemon";
```

Comment:
Should we move printing successful message logic to base class by making IEdgeDaemon as abstract class?

Reply:
I was considering doing that, but had decided to wait. I'll take another look.

Reply:
Actually I'm thinking of breaking up IEdgeDaemon.InstallAsync into InstallAsync and ConfigureAsync because although Windows doesn't need to set up config.yaml the way Linux does (the PowerShell installer script does it for us), it _does_ potentially need to add the proxy information to config.yaml. When I do that, sharing the logging code in a base class makes more sense. Stay tuned...

Reply:
Nevermind...I left if as just InstallAsync because if I add ConfigureAsync it will have all that same parameters that InstallAsync has anyway.

Reply:
With my most recent changes to configure the proxy in config.yaml inside InstallAsync, the message printing code is subtly different between Windows and Linux. I'll keep it as it is for now.